ui/src/app/pages/analysis/new-notebook-modal.tsx

```diff
@@ -6,16 +6,17 @@ import {styles as headerStyles} from 'app/components/headers';
 import {RadioButton, TextInput, ValidationError} from 'app/components/inputs';
 import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
 import {TooltipTrigger} from 'app/components/popups';
+import {nameValidationFormat} from 'app/components/rename-modal';
 import {userMetricsApi} from 'app/services/swagger-fetch-clients';
 import {summarizeErrors} from 'app/utils';
 import {navigate} from 'app/utils/navigation';
 import {Kernels} from 'app/utils/notebook-kernels';
+import {ResourceType} from 'app/utils/resourceActions';
 
-
-import {FileDetail, Workspace} from 'generated/fetch';
+import {Workspace} from 'generated/fetch';
```

Comment:
Huh, where was this coming from before? I would think that the `workspace: Workspace` on line 19 wouldn't work without this

Reply:
i think it previously existed on line 15.

`import {FileDetail, Workspace} from 'generated/fetch';	`

Reply:
Aha! Thank you!